jenkins/check-for-build.jenkinsfile

```diff
@@ -8,6 +8,7 @@ pipeline {
     triggers {
         parameterizedCron '''
             H/10 * * * * %INPUT_MANIFEST=1.3.0/opensearch-dashboards-1.3.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
+            H/10 * * * * %INPUT_MANIFEST=1.2.5/opensearch-1.2.5.yml;TEST_MANIFEST=1.2.5/opensearch-1.2.5-test.yml;TARGET_JOB_NAME=distribution-build-opensearch
             H/10 * * * * %INPUT_MANIFEST=1.2.4/opensearch-1.2.4.yml;TEST_MANIFEST=1.2.4/opensearch-1.2.4-test.yml;TARGET_JOB_NAME=distribution-build-opensearch
```

Comment:
We should remove 1.2.4 from this cron. The version is already out we do not need to keep rebuilding it